internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -306,3 +340,46 @@ func (q amsQuotaService) DeleteQuota(subscriptionId string) *errors.ServiceError
 	}
 	return nil
 }
+
+func (q amsQuotaService) DeleteQuotaForBillingModel(subscriptionId string, kafkaBillingModel config.KafkaBillingModel) *errors.ServiceError {
+	if subscriptionId == "" {
```

Comment:
What's the reason this is checked?

Reply:
I think so. But this check was already done for `DeleteQuota` so I preferred to keep it.